macintacos/vspacecode-config.json

```
@@ -552,128 +572,449 @@
               "key": "l",
-              "name": "Toggle list",
+              "name": "Locate configured Go tools",
+              "type": "command"
+            },
+            {
+              "command": "go.tools.install",
+              "key": "i",
+              "name": "Install/update tools",
+              "type": "command"
+            }
+          ],
+          "key": "e",
+          "name": "Environment...",
+          "type": "bindings"
+        },
+        {
+          "bindings": [
+            {
+              "command": "editor.action.revealDefinition",
+              "key": "d",
+              "name": "Go to definition",
+              "type": "command"
+            },
+            {
+              "command": "editor.action.peekDefinition",
+              "key": "D",
+              "name": "Peek definition",
+              "type": "command"
+            },
+            {
+              "command": "references-view.findReferences",
+              "key": "R",
+              "name": "Find all references",
+              "type": "command"
+            },
+            {
+              "command": "editor.action.referenceSearch.trigger",
+              "key": "r",
+              "name": "Peek references",
+              "type": "command"
+            },
+            {
+              "command": "editor.action.goToTypeDefinition",
+              "key": "t",
+              "name": "Go to type definition",
+              "type": "command"
+            },
+            {
+              "command": "editor.action.peekTypeDefinition",
+              "key": "T",
+              "name": "Peek type definition",
+              "type": "command"
+            },
+            {
+              "command": "extension.dash.specific",
+              "key": "E",
+              "name": "Go to documentation in Dash",
+              "type": "command"
+            },
+            {
+              "command": "workbench.action.problems.focus",
+              "key": "e",
+              "name": "Go to errors/problems",
+              "type": "command"
+            },
+            {
+              "command": "workbench.action.gotoMethod",
+              "key": "m",
+              "name": "Go to method in file",
               "type": "command"
             },
             {
-              "command": "markdown.showPreview",
+              "command": "workbench.action.gotoSymbol",
+              "key": "i",
+              "name": "Find symbol in file",
+              "type": "command"
+            },
+            {
+              "command": "workbench.action.showAllSymbols",
+              "key": "I",
+              "name": "Find symbol in project",
+              "type": "command"
+            }
+          ],
+          "key": "g",
+          "name": "Go to...",
+          "type": "bindings"
+        },
+        {
+          "bindings": [
+            {
+              "command": "go.godoctor.extract",
+              "key": "f",
```

```suggestion
              "key": "e",
```

This reflects spacemacs key binding and stands for "extract"

---

Okay. I actually just realized that this command doesn't _really_ work in projects that use modules (see https://github.com/golang/vscode-go/issues/677), so let me see if I can get it to use the proper VSCode implementation.